core/src/test/scala/com/pingcap/tispark/tls/TiSparkTLSSuite.scala

```diff
@@ -88,4 +89,12 @@ class TiSparkTLSSuite extends BaseTiSparkTest {
     assert(2 == df.collect().head.get(0))
     assert("TiKV".equals(df.collect().head.get(1)))
   }
+
+  test("test get TiDB version with HTTPS") {
+    if (!TLSEnable) {
+      cancel
+    }
+    val tiSparkTeleInfo = TiSparkTeleInfo.getTiSparkTeleInfo()
```

Comment:
Could we test both jks and pem

Reply:
yes, we have already test both jks and pem in Github ActionTLS test. [here](https://github.com/pingcap/tispark/runs/6665044299?check_suite_focus=true) show `test TLS` (pem cert) and `test JKS` (jks test).

Reply:
Does action include scalaj. If not, we should test it here or add to action.